features/step_definitions/pod.rb

```diff
@@ -155,6 +155,9 @@
     @result = pod.wait_till_status(BushSlicer::Pod::SUCCESS_STATUSES, user, ready_timeout)
 
     unless @result[:success]
+      if @result[:matched_status] == :pending
+        pod.describe(user)
+      end
```

Comment:
Do we need to check status? Wouldn't it be useful to get pod description at failure whatever current status is?
Other than that we can certainly merge this.
I think that we can also, for clarifying intention, call like `pod.describe(user, quiet: false)`

Reply:
good point.  I found another similar `:pending` failure in pod-for-ping.  

There are probably more places where we should dump `describe pod`

Reply:
err well it looks like there is a message in the `status.conditions` array as seen in the pod-for-ping issues.

So maybe we could just dump the last `oc get pods` status.